Modules/Filtering/ImageIntensity/include/itkSymmetricEigenAnalysisImageFilter.h

```diff
@@ -106,6 +106,12 @@ class SymmetricEigenAnalysisFunction
       m_Calculator.SetOrderEigenValues(false);
     }
   }
+  void
+  SetOrderEigenValuesBy(EigenValueOrderEnum order)
+  {
+    this->OrderEigenValuesBy(order);
+  }
+
```

Comment:
Is this method tested in the test file?

Reply:
@jhlegarreta good catch! :eagle: :eye: I added a `ITK_TEST_SET_GET_VALUE` check and the missing `Get` method.

Reply:
Not sure if I see the additions in the file test beyond the change that was already present to adapt to the new constructor: maybe you did not commit those changes :thinking: ?